### PR TITLE
Further translation suggestions

### PR DESCRIPTION
**Pull request by odscrachel**
Changes to:

* make it clearer the bods-main project vs versioned project in transifex
* more info for building the preview docs for reviewers

### CODE REVIEW

File: translations.md [hunk 1]
```diff
@@ -270,11 +270,27 @@ Once you've got all your translations, you need to publish them. The process for
 
 These instructions were summarised from [Localization of Documentation](https://docs.readthedocs.io/en/stable/localization.html) in the readthedocs docs.
 
+## Creating a preview
+
+During the translation process, there will be points where it will be helpful to generate a preview to allow the translators and/or reviewers to see the translations in context. 
+
+### Local build and Github 
+
+1. If you haven't already, create a staging branch.
```
Comment (kd-ods) [inline translations.md:279]:
```suggestion
1. If there isn't already a translation staging branch, create one locally. Switch to work on the staging branch.
```

File: translations.md [hunk 2]
```diff
@@ -270,11 +270,27 @@ Once you've got all your translations, you need to publish them. The process for
...
 These instructions were summarised from [Localization of Documentation](https://docs.readthedocs.io/en/stable/localization.html) in the readthedocs docs.
 
+## Creating a preview
+
+During the translation process, there will be points where it will be helpful to generate a preview to allow the translators and/or reviewers to see the translations in context. 
+
+### Local build and Github 
+
+1. If you haven't already, create a staging branch. 
+2. To fetch the current translations, run `tx pull -a` to fetch all, or `tx pull -l ru` to fetch a particular language. You may need to add a force flag (`tx pull -f -a` or `tx pull -f -l ru`)
```
Comment (kd-ods) [inline translations.md:280]: Rachel, I assume that the only dependency for this to all work fine is the .tx/config file being correctly set up and available on the branch? i.e. It doesn't matter if the pot files haven't been generated and are not available locally.

File: translations.md [hunk 3]
```diff
@@ -270,11 +270,27 @@ Once you've got all your translations, you need to publish them. The process for
...
+3. If the SVGs were translated, **build translated SVGs** for each language using itstool, and commit these (because we can't easily install itstool on readthedocs):
+  * Run `pybabel compile --use-fuzzy -d docs/locale -D svg`
+  * Replacing `<LANG>` with language code, eg, `ru` (run this once per language): `itstool -m docs/locale/<LANG>/LC_MESSAGES/svg.mo -o docs/_build_svgs/<LANG> docs/_assets/*.svg`
+4. Build the docs locally 
+  * Run `sphinx-build  -D language=ru  docs/ _build`
+  * Run `cd _build` to move to the build folder
+  * Run `python3 -m http.server` to check the build
+5. Add, commit and push your changes to the staging branch.
```
Comment (kd-ods) [inline translations.md:288]:
```suggestion
5. Add, commit and push your changes to the remote staging branch.
```

**Review by kd-ods — CHANGES_REQUESTED** [review]
Couple of suggestions and a question, @odscrachel .

File: translations.md (hunk 2 above)
Comment (odscrachel) [reply translations.md:280]: I assume so, providing that the branch you fork from has the pot files.

**Review by kd-ods — APPROVED** [review]
I'll merge this. Thanks, Rachel :-)

**Pull request merged**